schema/dereferenced-release-schema.json

```
@@ -2173,6 +2173,29 @@
                       "string",
                       "null"
                     ]
+                  },
+                  "weight": {
+                    "title": "Weight",
+                    "description": "The weight of the item unit.",
```

Unit occurs in a couple of places other than items (see https://github.com/open-contracting/standard/issues/1107).

```suggestion
                    "description": "The weight of one unit.",
```